Woof/Woof/Model/Walking/Reviews.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/**
+ The owner and sitter reviews for the walking service.
+ Conforms to the `Codable` protocol.
+ */
```

Comment:
Nit: I think we can not specify whose reviews is in structure.
```suggestion
/**
 The reviews for the walking service.
 */
```

Reply:
Fixed with 7f70e76